pkg/spdk/server.go

```diff
@@ -893,7 +893,11 @@ func (s *Server) EngineSnapshotCreate(ctx context.Context, req *spdkrpc.Snapshot
 		return nil, grpcstatus.Errorf(grpccodes.NotFound, "cannot find engine %v for snapshot creation", req.Name)
 	}
 
-	snapshotName, err := e.SnapshotCreate(spdkClient, req.SnapshotName)
+	opts := util.SnapshotOptions{
+		UserCreated: req.UserCreated,
+	}
+
+	snapshotName, err := e.SnapshotCreate(spdkClient, req.SnapshotName, opts)
```

Comment:
Why don't we use the same way for both replica and engine snapshot call? The engine call is using an option struct, but the replica call is just using a bool parameter.
> 	return r.SnapshotCreate(spdkClient, req.SnapshotName, req.UserCreated)

Can we just make them consistent?

Reply:
Done